Import html/semantics/forms/the-input-element WPTs 07/22/2025

rahimabdi opened the pull request:
#### 428c1c9b2f6d7480b37f48c803b72e6e8e589101
<pre>
Import html/semantics/forms/the-input-element WPTs 07/22/2025
<a href="https://bugs.webkit.org/show_bug.cgi?id=296361">https://bugs.webkit.org/show_bug.cgi?id=296361</a>

Reviewed by Tim Nguyen.

Import html/semantics/forms/the-input-element tests at upstream commit:
<a href="https://github.com/web-platform-tests/wpt/commit/70e23bfdc22a1c1d9bf46edf38c649533946c7b4">https://github.com/web-platform-tests/wpt/commit/70e23bfdc22a1c1d9bf46edf38c649533946c7b4</a>

This PR re-baselines imported WPT tests for platforms: ios, mac, gtk, wpe.

* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/color-attributes.window.html:
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/color.tentative-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/color.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/color.window.html:
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/input-valueasdate-typeerror-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/input-valueasdate-typeerror.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/input-valueasnumber-typeerror-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/input-valueasnumber-typeerror.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/number-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/number.html:
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-checked-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-checked.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-disabled-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-disabled.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-group-first-focus-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-group-first-focus.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-group-focusable-focus-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-group-focusable-focus.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focused-and-scrolled-into-view-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focused-and-scrolled-into-view.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/resources/focus-utils.js: Added.
(waitForRender):
(async pressKey):
(async arrowUp):
(async arrowDown):
(async navigateFocusForward):
(async navigateFocusBackward):
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/resources/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/show-picker-disabled-readonly.html:
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/w3c-import.log:
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/color.tentative-expected.txt: Added.
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-checked-expected.txt: Added.
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-disabled-expected.txt: Added.
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-group-first-focus-expected.txt: Added.
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-group-focusable-focus-expected.txt: Added.
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focused-and-scrolled-into-view-expected.txt: Added.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/color.tentative-expected.txt: Added.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-checked-expected.txt: Added.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-disabled-expected.txt: Added.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-group-first-focus-expected.txt: Added.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-group-focusable-focus-expected.txt: Added.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focused-and-scrolled-into-view-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/input-valueasdate-typeerror-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/show-picker-disabled-readonly-expected.txt:
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/color.tentative-expected.txt: Added.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-checked-expected.txt: Added.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-disabled-expected.txt: Added.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-group-first-focus-expected.txt: Added.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-group-focusable-focus-expected.txt: Added.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focused-and-scrolled-into-view-expected.txt: Added.

Canonical link: <a href="https://commits.webkit.org/298346@main">https://commits.webkit.org/298346@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/d2a876b1a7b5edddbeb3bc014f18adb0d796db70

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/115220 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/34923 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/25415 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/121328 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/65834 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/35574 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/43489 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/87544 "Passed tests") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/60/builds/42304 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/118168 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/28362 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/103436 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/67941 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/27534 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/21557 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/64978 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/97747 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/21673 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/124507 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/42175 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/31560 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/96345 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/42546 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/99626 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/96132 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/24456 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/41343 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/19189 "Passed tests") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/38134 "The change is no longer eligible for processing.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/42053 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/41579 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/44903 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/43307 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/97c6df5c18bf1a16561ae515aec53f4a34f2d0fc. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48404

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/1ef671ef80477c4253ef909f3d26c1104dee8140. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48404

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/400f474f0ffbbbc48c8305b7d84a8887a60b7ae7. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48404

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/2508b967fe8716f3d0f33a101fcbb5e17d222600. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48404

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/0afd806cbe21afb832f99e5c602f873243763ae4. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48404

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/0afd806cbe21afb832f99e5c602f873243763ae4. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48404

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/a2590d528c3e32a467817d3ef22a8db9b1bafbfc. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48404

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/8f0b8997d3009d46cf1b3d240445c70b26b71312. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48404

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/8f0b8997d3009d46cf1b3d240445c70b26b71312. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48404

rahimabdi commented:
@nt1m Could you please take another look? Thanks!

nt1m reviewed (commented):
2 general comments:
* Identical WPE / GTK baselines can be merged into a single one in GLIB (I haven't checked which ones specifically are identical)
* Why the ios-18 directory? Are the failures specific to an iOS version? They should be in ios if not

rahimabdi commented:
> * Identical WPE / GTK baselines can be merged into a single one in GLIB (I haven't checked which ones specifically are identical)

I put them in glib originally but it still failed unfortunately.

> * Why the ios-18 directory? Are the failures specific to an iOS version? They should be in ios if not

Yes, the ios bot was passing but ios18 failing. Thanks!

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/4ab6fabc31aa332287db69dc96ee506913b35960. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48404

rahimabdi committed:
Import html/semantics/forms/the-input-element WPTs 07/22/2025
https://bugs.webkit.org/show_bug.cgi?id=296361

Reviewed by Tim Nguyen.

Import html/semantics/forms/the-input-element tests at upstream commit:
https://github.com/web-platform-tests/wpt/commit/70e23bfdc22a1c1d9bf46edf38c649533946c7b4

This PR re-baselines imported WPT tests for platforms: ios, mac, gtk, wpe.

* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/color-attributes.window.html:
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/color.tentative-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/color.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/color.window.html:
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/input-valueasdate-typeerror-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/input-valueasdate-typeerror.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/input-valueasnumber-typeerror-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/input-valueasnumber-typeerror.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/number-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/number.html:
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-checked-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-checked.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-disabled-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-disabled.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-group-first-focus-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-group-first-focus.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-group-focusable-focus-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-group-focusable-focus.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focused-and-scrolled-into-view-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focused-and-scrolled-into-view.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/resources/focus-utils.js: Added.
(waitForRender):
(async pressKey):
(async arrowUp):
(async arrowDown):
(async navigateFocusForward):
(async navigateFocusBackward):
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/resources/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/show-picker-disabled-readonly.html:
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/w3c-import.log:
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/color.tentative-expected.txt: Added.
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-checked-expected.txt: Added.
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-disabled-expected.txt: Added.
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-group-first-focus-expected.txt: Added.
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-group-focusable-focus-expected.txt: Added.
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focused-and-scrolled-into-view-expected.txt: Added.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/color.tentative-expected.txt: Added.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-checked-expected.txt: Added.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-disabled-expected.txt: Added.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-group-first-focus-expected.txt: Added.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-group-focusable-focus-expected.txt: Added.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focused-and-scrolled-into-view-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/input-valueasdate-typeerror-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/show-picker-disabled-readonly-expected.txt:
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/color.tentative-expected.txt: Added.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-checked-expected.txt: Added.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-disabled-expected.txt: Added.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-group-first-focus-expected.txt: Added.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focus-navigation-group-focusable-focus-expected.txt: Added.
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/radio-focused-and-scrolled-into-view-expected.txt: Added.

Canonical link: https://commits.webkit.org/298346@main
